BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -329,6 +329,7 @@ extern NSString *const BOXAPIObjectKeyExtension;
 extern NSString *const BOXAPIObjectKeyIsPackage;
 extern NSString *const BOXAPIObjectKeyAllowedSharedLinkAccessLevels;
 extern NSString *const BOXAPIObjectKeyCollections;
+extern NSString *const BOXAPIObjectKeyCollectionMemberships;
```

Comment:
need to add this to the get all fields parameter as well

Reply:
done